src/compiler/checker.ts

```diff
@@ -5451,6 +5453,11 @@ namespace ts {
                                 localErrors = false;
                             }
                         }
+                        if (reportErrors) {

```

Comment:
For example [here](https://github.com/Microsoft/TypeScript/pull/5777/files#diff-4fd29482f7f7ab7fdaa86c7c8f945996R133). You've already reported why the signatures aren't compatible, so it doesn't make sense to say there isn't a corresponding signature.